device_doctor/test/src/host_utils_test.dart

```diff
@@ -52,10 +52,10 @@ void main() {
     });
 
     test('when there are unkilled processes', () async {
-      when(processManager.runSync(<String>['taskkill', '/pid', pid, '/f']))
-          .thenAnswer((_) => ProcessResult(1, 0, 'test', 'test'));
       pid = '123';
       output = '$pid abc';
+      when(processManager.runSync(<String>['taskkill', '/pid', pid, '/f']))
```

Comment:
This was the core issue. Strings are defined after the mock, so given a random order it occasionally chooses the wrong. This was causing 50% failures